content/components/table/rationale/alternating-rows.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: component/rationale.js
+heading: Alternating rows
+---
+
+For a data table with large amounts of rows, highlighting alternating rows helps [increase readability and scanning speed](http://alistapart.com/article/zebrastripingmoredataforthecase/). We've chosen for the bottom row to be highlighted using the `nth-last-child(odd)` selector, since this 'visually completes' the table. 
```

Comment:
"We've elected to highlight the bottom row using... to provide the table with visual completeness."